nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -373,6 +373,21 @@
 
 - [`system.stateVersion`](#opt-system.stateVersion) is now validated and must be in the `"YY.MM"` format, ideally corresponding to a prior NixOS release.
 
+- `services.mysql` now supports easy cluster setup via `services.mysql.galeraCluster` option.
```

Comment:
```suggestion
- `services.mysql` now supports easy cluster setup via [`services.mysql.galeraCluster`](#opt-services.mysql.galeraCluster.enable) option.
```
Most entries link to new options. LGTM otherwise

Reply:
thanks :)